com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl

```diff
@@ -26,10 +26,12 @@
             output.screenUV = ComputeScreenPos(clipVertex).xy; \
             output.positionWS = worldSpacePos;
 
+//half3 normalUnpacked = UnpackNormal(normal);
         #define APPLY_NORMALS_LIGHTING(input, lightColor)\
             half4 normal = SAMPLE_TEXTURE2D(_NormalMap, sampler_NormalMap, input.screenUV);\
-            half3 normalUnpacked = UnpackNormal(normal);\
+            half3 normalUnpacked;\
             half3 dirToLight;\
+            normalUnpacked.xyz = 2 * normal.rgb - 1;\
```

Comment:
We should use UnpackNormalRGBNoScale(), which does exactly the same calculation.

Reply:
Also, the same change should be applied to the other branch of the #ifdef.

Reply:
Fixed